bids/layout/validation.py

```diff
@@ -126,7 +126,7 @@ def check_for_description(bids_dir):
                                   if sd.is_dir() and check_for_description(sd))
 
     if not deriv_dirs:
-        warnings.warn("Derivative indexing was requested, but no valid "
+        warnings.warn("Derivative indexing was requested, but derivatives were not valid "
                         "datasets were found in the specified locations "
```

Comment:
This reads oddly to me. Would the following modification clarify things?

```suggestion
        warnings.warn("Derivative indexing was requested, but no valid "
                        "derivative datasets were found in the specified locations "
```

Reply:
works for me!